src/app/cli/src/commands/system_diagnose_command.rs

```diff
@@ -0,0 +1,197 @@
+
+    async fn run_container_rootless_check(&self, f: &mut Stdout) -> Result<(), CLIError> {
+        write!(f, "\ncontainer rootless check... ")?;
+
+        let is_container_rootless = match self
+            .container_runtime
+            .run_attached(DUMMY_IMAGE)
+            .init(true)
+            .spawn()
+            .unwrap()
+            .wait()
+            .await
+        {
+            Ok(_) => style("ok").green(),
+            Err(_) => style("failed").red(),
+        };
+
+        write!(f, "{is_container_rootless}")?;
+        Ok(())
+    }
+
+    async fn run_container_volume_mount_check(&self, f: &mut Stdout) -> Result<(), CLIError> {
+        write!(f, "\ncontainer volume mounts work... ")?;
+        let temp_dir = tempfile::tempdir()?;
+        let cwd = Path::new(".").canonicalize()?;
+
+        let file_path = temp_dir.path().join("tmp.txt");
+        let _ = File::create(file_path)?;
+
+        let is_container_rootless = match self
+            .container_runtime
+            .run_attached(DUMMY_IMAGE)
+            .volume((cwd, temp_dir.into_path()))
+            .init(true)
+            .spawn()
+            .unwrap()
+            .wait()
+            .await
+        {
+            Ok(_) => style("ok").green(),
+            Err(_) => style("failed").red(),
+        };
+
+        write!(f, "{is_container_rootless}")?;
+        Ok(())
+    }
+
+    async fn run_workspace_consistency_check(
+        &self,
+        dataset_repo: Arc<dyn DatasetRepository>,
+        verification_svc: Arc<dyn VerificationService>,
+        f: &mut Stdout,
+    ) -> Result<(), CLIError> {
+        write!(f, "\nworkspace consistent... ")?;
+        let datasets: Vec<_> = dataset_repo.get_all_datasets().try_collect().await?;
+
+        let verify_options = VerificationOptions {
+            check_integrity: true,
+            replay_transformations: false,
+        };
+        for dataset in datasets {
+            if let Err(_) = verification_svc
+                .verify(
+                    &dataset.as_local_ref(),
+                    (None, None),
+                    verify_options.clone(),
+                    None,
+                )
+                .await
+            {
+                write!(f, "{}", style("failed").red())?;
+                return Ok(());
+            };
+        }
+        write!(f, "{}", style("ok").green())?;
+
+        Ok(())
+    }
+
+    pub async fn run(
+        &self,
+        dataset_repo: Arc<dyn DatasetRepository>,
+        verification_svc: Arc<dyn VerificationService>,
+    ) -> Result<(), CLIError> {
+        let mut out = std::io::stdout();
+
+        self.run_container_check(&mut out).await?;
+        self.run_container_pull_check(&mut out).await?;
+        self.run_container_rootless_check(&mut out).await?;
+        self.run_container_volume_mount_check(&mut out).await?;
+        self.run_workspace_consistency_check(dataset_repo, verification_svc, &mut out)
+            .await?;
+        Ok(())
```

Comment:
Here is what it prints - the last reported line interfers with the next shell prompt:

```
sergeiz@ubuntu:~/kamu/testing/test3$ kamu system diagnose
container installed... ok
container can pull images... ok
container rootless check... ok
container volume mounts work... ok
workspace consistent... oksergeiz@ubuntu:~/kamu/testing/test3$
```

Reply:
Plz improve the messages, so it's clearer which container system is queried.